google-cloud-spanner/src/main/java/com/google/cloud/spanner/BackupInfo.java

```diff
@@ -28,8 +28,22 @@
 
     abstract Builder setSize(long size);
 
+    /**
+     * Output only.
```

Comment:
What does this mean?

Reply:
Updated the comment to the proto comment.